cmd/kind/load/image-archive/image-archive.go

```diff
@@ -99,28 +98,9 @@ func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
 
 	// Load the image into every node
 	for _, node := range selectedNodes {
-		if err := loadImage(args[0], &node); err != nil {
+		if err := loadutils.LoadImage(args[0], &node); err != nil {
```

Comment:
side note: do we need to check if `len(args) > 0`?
[1]

Reply:
not really, the `NewCommand` func already does this:
```go
cmd := &cobra.Command{                                                      
    Args: func(cmd *cobra.Command, args []string) error {                   
        if len(args) < 1 {                                                  
            return errors.New("name of image archive is required")          
        }
    ...                                                                   
```

Reply:
Why not simply using `cobra.MinimumNArgs(int)` ?